src/images/icons/collapse.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+    <title>
```

Comment:
@jakpaw Some time ago we decided to use `aria-labelledby` to name icons (instead of `<title/>`) but forgot to update the instruction. Could you please remove `<title/>` from all svgs?